packages/common/src/types.ts

```diff
@@ -46,7 +46,7 @@ export interface GenesisBlockConfig {
 
 export interface HardforkConfig {
   name: Hardfork | string
-  block?: number | null
+  block: number | null // null is used for hardforks that should not be applied -- since `undefined` isn't a valid value in JSON
```

Comment:
If I don't want to apply the hf, then I can also just not create the object itself, right?

Reply:
Yes.

Reply:
Yes, we mainly have this option in to allow for defining a future HF like `shanghai` and already be able to switch to this HF in a static context and so activate the HF rules without knowing the block number (yet).